web/views.py

```diff
@@ -319,7 +319,7 @@ def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
         if self.request.user.is_authenticated:
             context['has_demographic'] = self.request.user.latest_demographics
-            context['children'] = self.request.user.children.all()
+            context['children'] = self.request.user.children.filter(deleted = False)
```

Comment:
It really doesn't matter but if you're fixing the x and the anchor thing would you kill the spaces around the `=`? Thanks!